__tests__/integration/hathorwallet_facade.test.js

```diff
@@ -354,72 +375,66 @@ describe('getFullHistory', () => {
     );
     await waitForTxReceived(hWallet, rawMoveTx.hash);
 
-    let history = hWallet.getFullHistory();
+    const history = hWallet.getFullHistory();
     expect(Object.keys(history)).toHaveLength(2);
     expect(history).toHaveProperty(rawMoveTx.hash);
     const moveTx = history[rawMoveTx.hash];
 
     // Validating transactions properties were correctly translated
-    expect(moveTx).toHaveProperty('tx_id', rawMoveTx.hash);
-    expect(moveTx).toHaveProperty('version', rawMoveTx.version);
-    expect(moveTx).toHaveProperty('weight', rawMoveTx.weight);
-    expect(moveTx).toHaveProperty('timestamp', rawMoveTx.timestamp);
-    expect(moveTx).toHaveProperty('is_voided', false);
-    expect(moveTx.parents).toEqual(rawMoveTx.parents);
+    expect(moveTx).toMatchObject({
+      tx_id: rawMoveTx.hash,
+      version: rawMoveTx.version,
+      weight: rawMoveTx.weight,
+      timestamp: rawMoveTx.timestamp,
+      is_voided: false,
+      parents: rawMoveTx.parents,
+    });
 
     // Validating inputs
     expect(moveTx.inputs).toHaveLength(rawMoveTx.inputs.length);
     for (const inputIndex in moveTx.inputs) {
-      const inputObj = moveTx.inputs[inputIndex];
-
-      // Some attributes were correctly translated
-      expect(inputObj.index).toEqual(rawMoveTx.inputs[inputIndex].index);
-      expect(inputObj.tx_id).toEqual(rawMoveTx.inputs[inputIndex].hash);
-
-      // Decoded attributes are correct
-      expect(inputObj).toHaveProperty('token', HATHOR_TOKEN_CONFIG.uid);
-      expect(inputObj).toHaveProperty('token_data', TOKEN_DATA.HTR);
-      expect(inputObj).toHaveProperty('script');
-      expect(inputObj).toHaveProperty('value', 10);
-      expect(inputObj).toHaveProperty('decoded');
-      expect(inputObj.decoded).toHaveProperty('type', 'P2PKH');
-      expect(inputObj.decoded).toHaveProperty('address', fundDestinationAddress);
+      expect(moveTx.inputs[inputIndex]).toMatchObject({
+        // Some attributes were correctly translated
```

Comment:
Which attributes were not correctly translated?

Reply:
Re-worded this on [dc82fee](https://github.com/HathorNetwork/hathor-wallet-lib/pull/393/commits/dc82fee1edfeb1867d4e95902343c36393456b22)